app/controllers/phases_controller.rb

```diff
@@ -162,6 +162,10 @@ def admin_create
     @phase.description = params["phase-desc"]
     @phase.modifiable = true
     if @phase.save
```

Comment:
actually after switching to callbacks it doesn't really work due to the `deep_copy` methods since they're saving the cloned object before the template association has been updated. I've made a note in #170 about moving these functions afterward. I think much of the publish/customize logic could be moved to callbacks on the template model as well.